Photographers/tests/test_photographers_site.py

```diff
@@ -7,13 +7,162 @@
+
+
+def test_about_page_title_visible(create_driver):
+    driver = create_driver
+    about_project_page = MainPage(driver).click_about_project_link()
+    assert about_project_page.title_page_visible()
+
+
+def test_photograph_link_is_visible(create_driver):
+    driver = create_driver
+    photo_of_the_day = MainPage(driver).click_photo_of_the_day().dropdown_click()
+    assert photo_of_the_day.find_photographs_link(), "Link photographer is not presented"
+
+
+def test_model_link_is_visible(create_driver):
+    driver = create_driver
+    photo_of_the_day = MainPage(driver).click_photo_of_the_day().dropdown_click()
+    assert photo_of_the_day.find_model(), "Link model is not presented"
+
+
+def test_find_pro_photographer_link_is_visible(create_driver):
+    driver = create_driver
+    photo_of_the_day = MainPage(driver).click_photo_of_the_day().dropdown_click()
+    assert photo_of_the_day.find_prof_photographer(), "Link PRO photographer is not presented"
+
+
+def test_login_by_google(create_driver):
+    driver = create_driver
+    registration_page = MainPage(driver).click_registration_link()
+    assert registration_page.login_by_google_visible(), "Link fo registration by Google is not presented"
+
+
+def test_login_by_facebook(create_driver):
+    driver = create_driver
+    registration_page = MainPage(driver).click_registration_link()
+    assert registration_page.login_by_facebook_visible(), "Link fo registration by FaceBook is not presented"
+
+
+def test_ingo_block_at_the_page(create_driver):
+    driver = create_driver
+    registration_page = MainPage(driver).click_registration_link()
+    assert registration_page.why_register_block_visible(), "Information block is not presented"
+
+
+def test_check_next_month_link(create_driver):
+    driver = create_driver
+    top_100_photographers_page = MainPage(driver).move_top_dropdown().click_top_100_photographers()
+    assert top_100_photographers_page.check_link_is_not_at_the_page() is True, "The link is  at the page for" \
+                                                                               " future month"
+
+
+def test_check_first_month_results(create_driver):
+    driver = create_driver
+    top_100_photographers_page = MainPage(driver).move_top_dropdown().click_top_100_photographers()
+    assert top_100_photographers_page.check_search_by_months(), "January is in the list"
+
+
+def test_switch_to_eng_link(create_driver):
+    driver = create_driver
+    main_page = MainPage(driver)
+    assert main_page.find_eng_switch(), "Link for switch language is at the page"
+
+
+def test_switch_to_eng(create_driver):
     driver = create_driver
-    main_page = MainPage(driver).click_user_icon().fill_email(email).fill_pass.click_login().click_burger_menu()
+    main_page = MainPage(driver).click_eng_switch()
+    assert main_page.check_gallery_element_visible_text() == "Gallery", "Not English text at the site"
```

Comment:
Why do you have all tests in one file? 
Where are marks for the tests as described in task?